### DISCUSSION TITLE
Invalid checksum

### DISCUSSION THREAD

**Discussion by tdarcy01** [UglyToad/PdfPig#612] [Q&A]
Hi,
I'm getting this in my test output each time i run a test that verifies pdf files using PdfPig (latest version)
` Debug Trace:
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.
testhost.net472.x86.exe Warning: 0 : Table with invalid checksum found in TrueType font file.`

Very annoying. Any ideas how to fix this or disable warnings?

Thank you

**Comment by mischiminator** [comment]
i have the same problem, is there an option to disable this behavior or redirect the message to somewhere (using a custom logger doesn't help)

Edit: i just compiled the project myself with the line commented out :D

**Comment by refex** [comment]
so?
